apps/slash-stories/src/Layout/HeaderTitle/HeaderTitle.stories.tsx

```diff
@@ -69,3 +89,42 @@ export const ComplexTitle: StoryObj<typeof HeaderTitle> = {
     ),
   },
 };
+
+export const ComplexTitleWithContentAndChildren: StoryObj<typeof HeaderTitle> =
+  {
+    args: {
+      contentLeft: (
+        <div className="af-title-bar__actions">
+          {leftActions.map(
+            ({ icon, id, title }: ComponentProps<typeof Action>) => (
+              <a
+                key={id}
+                href="/#"
+                role="button"
+                title={title}
+                className="btn  af-btn--circle"
+              >
+                <Svg src={icon} />
```

Comment:
as it is just a story i dont think it is useful but:

svg is just for presentation so it should have the `role="presentation"`